.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: generic
+jobs:
+  include:
+    - stage: Test container build
+      before_install: pip install --user awscli
+      script: make dist
+    - stage: Deploy staging
+      install: pip install --user awscli
+      script: skip
+      before_deploy: make dist
+      deploy: 
```

Comment:
Is travis magically just only running this on master branch changes?

Reply:
Never mind, found in the docs it only runs on master unless another branch is supplied. 🌈